doc.go

```diff
@@ -27,19 +27,28 @@
 //
 // Status
 //
-// Alpha. Expect minor API changes and bug fixes. (Beta release coming soon)
+// Beta. Expect minor API changes and bug fixes. See our changelog (CHANGELOG.md)for more.
+//
 //
 // What's included
 //
 // UberFx builds the following into your service:
-// * Configuration, logging, metrics, and tracing setup by default
-// * Configuration provider that seamlessly merges static and dynamic config
-// * Application-level as well as runtime metrics for effective monitoring
-// * Context-aware logging for easy debugging
-// * RPC module with Thrift interfaces for microservices
-// * HTTP module with intelligent defaults for web applications
-// * Task module for executing async tasks durably
 //
+// • Logging backed by the zap logger
```

Comment:
![image](https://cloud.githubusercontent.com/assets/199077/23148510/1cbf8604-f79b-11e6-9ccd-3e4367ba70de.png)

Looks like file diff shows the wrong symbol, but the conversation view is fine :/